### DISCUSSION TITLE
[zksync-cli] <Title> Error Encountered: Network does not support ENS in zksync-cli transfer command

### DISCUSSION THREAD

**Discussion by samui68400** [zkSync-Community-Hub/zksync-developers#420] [General] [answered]
### Environment

Testnet

### Provide a brief description of the functionality you're trying to implement.

I'm encountering an issue while executing the transfer command with zksync-cli. When attempting to transfer funds to a specific Ethereum address, I'm receiving the following error: "network does not support ENS".

### What is the specific issue or error you're encountering?

ⓘ There was an error while doing transfer
ⓘ network does not support ENS (operation="getResolver", network="unknown", code=UNSUPPORTED_OPERATION, version=providers/5.7.2)

### Can you share the error messages or logs you're receiving, if any?

Execute the command 

**npx zksync-cli wallet transfer --to ADDRESS --amount 0.001 --token ETH --pk PRIVATEKEY --chain zksync-sepolia**

Observe the error message mentioned above.

### Have you made any recent changes to the contract before encountering this issue?

This issue occurs consistently whenever attempting to transfer funds using zksync-cli.
I haven't made any recent changes to the contract or environment before encountering this issue.
I'm using the latest version of zksync-cli.

### Are there any external libraries or contracts that your contract interacts with?

No, the issue is related to the zksync-cli tool provided by zkSync for interacting with the zkSync protocol.

### Can you provide the relevant portions of your contract code where the issue is occurring?

There is no contract code involved in this issue. The problem arises when executing commands with zksync-cli.

### Have you tried to isolate the problem, and if so, what were the results?

Yes, I attempted to isolate the problem by executing the transfer command with different parameters, but the issue persisted.

### What steps have you already taken to try to resolve the issue?

I have reviewed the zkSync documentation and searched for similar issues on forums and GitHub repositories. However, I haven't found a solution yet.

### Repo Link (Optional)

N/A (There's no specific repository associated with this issue)

### Additional Details

I'm using the latest version of zksync-cli and have encountered this issue consistently when attempting to transfer funds. Any assistance in resolving this issue would be greatly appreciated.

**Comment by JackHamer09** [comment, marked as answer]
This issue happens because of the `--token ETH` option in your command.
`--token` option expects token address instead of symbol. ETH will also be used by default if `--token` option is not specified.

Please see the [documentation](https://docs.zksync.io/build/tooling/zksync-cli/commands/wallet.html#wallet-transfer) for usage instructions

**Comment by samui68400** [comment]
Thanks for the answer
